README.md

```diff
@@ -60,15 +60,15 @@ Ruby is a backend, object-oriented programming language. Here you will learn the
 Rails is full stack framework built into the Ruby programming language. Here you will be introduced to Rails concepts and persistence storage with Postgres.
 
 1) [Rails Intro to Routes, Controllers and View](./rails/01rails_routes_controllers_views.md)
```

Comment:
This one needs an update too. Note: It references rails folder and not RailsC&V

Reply:
Good catch.  Will you check one more time for me?

Reply:
Good to go!